pandas/io/excel/_base.py

```diff
@@ -508,9 +508,7 @@ class ExcelWriter(object):
         Format string for datetime objects written into Excel files
         (e.g. 'YYYY-MM-DD HH:MM:SS')
     mode : {'w' or 'a'}, default 'w'
-        File mode to use (write or append).
-
-    .. versionadded:: 0.24.0
+        File mode to use (write or append). .. versionadded:: 0.24.0
```

Comment:
is there a reason for this? Not sure if this will render correctly in the html

Reply:
Yeah, that's right. I was about to fix it.